pkg/infra/pulumi_aws/iac/eks.ts

```diff
@@ -117,13 +123,14 @@ interface EksClusterOptions {
     initializePluginsOnFargate?: boolean
     installPlugins?: plugins[]
     enableFargateLogging?: Boolean
+    enableFluentBitLogging?: Boolean
     adminRoleArn?: string
     autoApply?: boolean
     createNodeGroup?: boolean
 }
 
 export const DefaultEksClusterOptions: EksClusterOptions = {
-    initializePluginsOnFargate: true,
+    initializePluginsOnFargate: false,
```

Comment:
What's the impact of this change?

Reply:
the alb controller and coredns will be run on our node group we create instead of fargate now. We always created the nodegroup so was just added cost having them be on fargate